### ISSUE TITLE
Itemshop not accepting items inside duplicates

### ISSUE THREAD

**Issue by Artuvazro** [rathena/rathena#4847]
* **rAthena Hash**:  05ebde01

* **Client Date**:  2015-05-13 and 2018-06-21

* **Server Mode**: pre-renewal

* **Description of Issue**: 
  * Result:  Itemshop should let you use items as currency. Currently it displays the following error: "You do not have enough Kafra Credit Points" and doesn't let you buy anything from it.
  * Expected Result:  The shop should allow buying items using another item as currency.
  * How to Reproduce: 

```
-	itemshop	IShop	91,512,12208:5

prontera,59,171,4	duplicate(IShop)	Item Shop#i_shop	91
```

**Comment by Artuvazro** [comment]
Update:
It doesn't work inside duplicates. But if you declare the shop npc in the same line, it works. I'm not closing the issue until someone confirms if this is intended or not.

**Comment by Daegaladh** [comment]
Fixed on fe14af99625d16f3c7ef1e8192a2d3b0214f33f1
